routes/RoutesConfig.jsx

```diff
@@ -1,11 +1,13 @@
 import React from "react";
 import { BrowserRouter, Routes as RouterRoutes, Route } from "react-router-dom";
 
-import HomePage from "@pages/Dashboard";
-import NotFoundPage from "@pages/NotFoundPage";
-import Login from "@pages/Registration/Login";
-import ForgotPassword from "@pages/Registration/ForgotPassword";
-import Landing from "@pages/Landing";
+import { HomePage } from "@pages/Dashboard";
+import { NotFoundPage } from "@pages/NotFoundPage";
+import { Login } from "@pages/Registration/Login";
+import { ForgotPassword } from "@pages/Registration/ForgotPassword";
+import { Landing } from "@pages/Landing";
+import { StudentList } from "@pages/Student/List";
+import { CreateNewStudent } from "@pages/Student/CreateNew";
```

Comment:
always load your pages lazy. please go through [Lazy Loading and code splitting](https://react.dev/reference/react/lazy)